pombola/south_africa/static/sass/_south-africa_overrides.scss

```diff
@@ -555,6 +573,32 @@ menu#mobile-top-tools,
     }
 }
 
+@media only all and (min-width: 640px) {
+    .about-this-site-menu__facebook,
+    .about-this-site-menu__twitter {
+        position: absolute;
+        top: (114px / 2) - (38px / 2);
```

Comment:
Thanks for explaining this in IRC - it'd be great if you could add a comment explaining where the 114px and 38px values come from, and that this is vertically centring the icon in the header...